### PR TITLE
detail build steps in README

### PR DESCRIPTION
**Pull request by vorburger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,11 @@
 
 First, install all [build dependencies](docs/dev/dependencies.md).
 
-Clone this repository to `src/github.com/openshift/installer` in your [GOPATH](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable). Then build the `openshift-install` binary with:
+Clone this repository to `src/github.com/openshift/installer` in your [GOPATH](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable) and build the `openshift-install` binary with `hack/build.sh`, so:
 
 ```sh
+mkdir $GOPATH/src/github.com/openshift ; cd $GOPATH/src/github.com/openshift
```
Comment (abhinavdahiya) [inline README.md]: So building installer from source assumes basic go knowledge. and i think the text above is adequate to inform developer to clone the repo to correct location and run `hack/build.sh`

File: README.md [hunk 2]
```diff
@@ -10,9 +10,11 @@
 
 First, install all [build dependencies](docs/dev/dependencies.md).
 
-Clone this repository to `src/github.com/openshift/installer` in your [GOPATH](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable). Then build the `openshift-install` binary with:
+Clone this repository to `src/github.com/openshift/installer` in your [GOPATH](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable) and build the `openshift-install` binary with `hack/build.sh`, so:
```
Comment (abhinavdahiya) [inline README.md]: i do not think we need to repeat the `hack/build.sh` twice.

File: README.md (hunk 1 above)
Comment (vorburger) [reply README.md]: so want me to abandon this? just checking.

**Comment by abhinavdahiya** [comment]
This looks stale, please feel free to re-open.

/close

**Pull request closed without merging**